tests/basic/task.yaml

```diff
@@ -29,4 +29,10 @@ execute: |
   test -f ${rootfs_folder}/etc/ssl/openssl.cnf
   test -f ${rootfs_folder}/usr/lib/*-linux-*/libssl.so.*
   test -f ${rootfs_folder}/etc/ld.so.conf.d/*-linux-*.conf
-  test -f ${rootfs_folder}/lib/*-linux-*/libc.so.*
+  if [[ "${RELEASE}" == "24.04" ]]
+  then
+    libc_base_path="/usr/lib/"
+  else
+    libc_base_path="/lib/"
+  fi
+  test -f ${rootfs_folder}${libc_base_path}*-linux-*/libc.so.*
```

Comment:
[Nit] I think it is clearer if you leave the slash outside the variable so:
```bash
 if [[ "${RELEASE}" == "24.04" ]]
  then
    libc_base_path="/usr/lib"
  else
    libc_base_path="/lib"
  fi
test -f ${rootfs_folder}${libc_base_path}/*-linux-*/libc.so.*
```

Reply:
why not :) -> https://github.com/canonical/chisel/pull/121/commits/9f51a364f7a1a48992a2ec3757b6dcd450aa423f